pkgs/racket-doc/scribblings/reference/contracts.scrbl

```diff
@@ -1918,7 +1916,8 @@ is bound to vectors of two elements, the exported identifier and a
 syntax object for the expression that produces the contract controlling
 the export.
 
-@history[#:changed "7.3.0.3" @list{Added @racket[#:unprotected-submodule].}]
+@history[#:changed "7.3.0.3" @list{Added @racket[#:unprotected-submodule].}
+         #:changed "7.7.0.9" @list{Deprecated @racket[ignored-id].}]
```

Comment:
Maybe "started ignoring @racket[ignored-id]" instead of "deprecated"

Reply:
Should I change the prose as well? I use the word "deprecate" there too -- I mean, I'm not clear what do you want to do.

Reply:
Deprecate is a fine word for the current (post-PR) state, but thing that changed is that ignored-id is now ignored. So I suggest changing only the history annotation.

It is a minor point for sure :)